### PR TITLE
Add reviewers

### PR DESCRIPTION
**Pull request by sdodson**
/cc @abhinavdahiya

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -11,6 +10,10 @@ aliases:
     - wking
   installer-reviewers:
     - vikramsk
```
Comment (abhinavdahiya) [inline OWNERS_ALIASES]: we should drop `vikramsk`

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -11,6 +10,10 @@ aliases:
...
+    - sdodson
```
Comment (abhinavdahiya) [inline OWNERS_ALIASES]: and lets keep them alpha sorted

File: OWNERS_ALIASES (hunk 1 above)
Comment (sdodson) [reply OWNERS_ALIASES]: dropped

**Pull request merged**